aws/src/main/python/idleNotifier/lambda_function.py

```diff
@@ -24,13 +24,18 @@ def lambda_handler(event, context):
         instance_name = "NA"
     subject = get_subject_from(event)
     logger.info('instance: ' + instance + '; region: ' + region + '; subject:' + subject)
+    email=get_email_from(event)
+    user_slacks_ids=os.environ['USER_SLACK_IDS']
+    slack_ids_as_dict = json.loads(user_slacks_ids)
+    channel_id=safe_value_with_default(slack_ids_as_dict, email, 'channel')
+
     payload = {
         "blocks": [
             {
                 "type": "section",
                 "text": {
                     "type": "mrkdwn",
-                    "text": f"*EC2 Idle Alarm Triggered*\n> *Alarm Name*\n> {alarm_name}\n> *Instance Name*\n> {instance_name}\n> *Trigger Subject*\n> {subject}\n> *Link to Alarm*\n> https://console.aws.amazon.com/cloudwatch/home?region={region}#alarmsV2:alarm/{alarm_name}\n> *Link to Instance*\n> https://console.aws.amazon.com/ec2/home?region={region}#Instances:instanceId={instance}"
+                    "text": f"*<@{channel_id}|cal> EC2 Idle Alarm Triggered*\n> *Alarm Name*\n> {alarm_name}\n> *Instance Name*\n> {instance_name}\n> *Trigger Subject*\n> {subject}\n> *Link to Alarm*\n> https://console.aws.amazon.com/cloudwatch/home?region={region}#alarmsV2:alarm/{alarm_name}\n> *Link to Instance*\n> https://console.aws.amazon.com/ec2/home?region={region}#Instances:instanceId={instance}"
```

Comment:
Notes for myself after this has been running:
* @here is really !here - and sometimes this is a miss (still diagnosing)
* The email is ONLY filled in for when we send the JSON, not when it comes from SNS (Checking for a solution to store in the alarm - short of duplicating checking direct from the instance)